perf/istio-install/values-new-installer.yaml

```diff
@@ -0,0 +1,134 @@
+# This is used to generate istio.yaml for a stress/load test cluster.
+
+global:
+  telemetryNamespace: istio-telemetry
+  configNamespace: istio-control
+  istioNamespace: istio-system
+  policyNamespace: istio-policy
+
+  imagePullPolicy: Always
+  refreshInterval: 30s
+  k8sIngressSelector: ingressgateway
+
+  controlPlaneSecurityEnabled: true
+
+  proxy:
+    enableCoreDump: true
+    concurrency: 2
+    accessLogFile: ""
+
+gateways:
+  istio-ilbgateway:
+    enabled: true
+
+  istio-ingressgateway:
+    enabled: true
+    secretVolumes:
+      - name: istio-ingressgateway-certs
+        secretName: istio-ingressgateway-certs
+        mountPath: /etc/istio/ingressgateway-certs
+      - name: istio-ingressgateway-certs-fortiotls
+        secretName: istio-ingressgateway-certs-fortiotls
+        mountPath: /etc/istio/ingressgateway-certs-fortiotls
+      - name: istio-ingressgateway-certs-fortionoistio
+        secretName: istio-ingressgateway-certs-fortionoistio
+        mountPath: /etc/istio/ingressgateway-certs-fortionoistio
+    labels:
+      istio: ingressgateway
+      ver: ingress10
+      app: istio-ingressgateway
+      name: gateway
+    type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
+    ports:
+      - port: 80
+        targetPort: 80
+        name: http2
+        nodePort: 31380
+      - port: 443
+        name: https
+        nodePort: 31390
+      - port: 31400
+        name: tcp
+      - port: 5201
+        name: tcp-iperf-cont
+      - port: 5202
+        name: tcp-iperfraw
+      - port: 5203
+        name: tcp-iperf
+      - port: 5204
+        name: tcp-iperf-tls
+
+pilot:
+  env:
+    PILOT_PUSH_THROTTLE: 50
+    GODEBUG: gctrace=1
+
+ingress:
+  enabled: false
+
+sidecar-injector:
+  enabled: true
+
+grafana:
+  enabled: true
+  prometheusNamespace: istio-prometheus
+  # datasources:
+  #   datasources.yaml:
+  #     apiVersion: 1
+  #     datasources:
+  #       - name: Prometheus
+  #         type: prometheus
```

Comment:
how come this is commented out?

Reply:
I changed this to use the built in support for prometheus operator that Doug added to the new installer, so now we can use the default datasource that comes with the grafana install.

Reply:
I cannot deploy Prometheus by running ./setup_istio_release.sh, could this be the reason?